times.py

```diff
@@ -337,8 +338,8 @@ def calculate(dirname: str, tipe: Type, product: str, formatter: Callable[[Mappi
     result: str = "NA"
     try:
       result = str(runtime.time())
-    except ValueError:
-      pass
+    except ValueError as ve:
+      sys.stderr.write('E: ValueError - ' + str(ve) + '\n')
```

Comment:
I think it'd make more sense to use [`warnings.warn`](https://docs.python.org/3/library/warnings.html#warnings.warn) here instead of `sys.stderr` directly. This is more flexible and would allow us to do things later like suppress warnings, upgrade them into full exceptions, or pipe the warnings to a more-elaborate logger (via `warnings.showwarning`).

Reply:
I wasn't familiar with that (wow, Python has everything) – sgtm.